tomcat/base.go

```diff
@@ -331,7 +331,14 @@ func (b Base) ContributeLogging(layer libcnb.Layer) error {
 
 	b.Logger.Bodyf("Writing %s/bin/setenv.sh", layer.Path)
 
-	s := fmt.Sprintf(`CLASSPATH="%s"`, file)
+	var s string
+	additionalJars, ok := os.LookupEnv("BPI_TOMCAT_ADDITIONAL_JARS")
+	if ok {
+		b.Logger.Bodyf("found BPI_TOMCAT_ADDITIONAL_JARS %q", additionalJars)
+		s = fmt.Sprintf(`CLASSPATH="%s:%s"`, file, additionalJars)
+	} else {
+		s = fmt.Sprintf(`CLASSPATH="%s"`, file)
+	}
```

Comment:
```suggestion
	s := fmt.Sprintf(`CLASSPATH="%s"`, file)
	if additionalJarsToClasspath, ok := os.LookupEnv("CLASSPATH"); ok {
		s = fmt.Sprintf(`CLASSPATH="%s:%s"`, file, additionalJarsToClasspath)
	}
	b.Logger.Debugf("Setting %s", s)
```

Reply:
* Regarding the name of the environment variable, I think that `CLASSPATH` is rather misleading. It is only a builpdack internal one.
* Regarding the `if/else` vs. `if`, that is a style question imho. I original had if like this, but changed it after [this](https://github.com/paketo-buildpacks/apache-tomcat/pull/375/files#r1328211642) comment.

Reply:
> Regarding the if/else vs. if, that is a style question imho. I original had if like [this](https://github.com/paketo-buildpacks/apache-tomcat/pull/375/files#r1328211642), but changed it after this comment.

Typically idiomatic go says that if you're setting a variable to based on the if block, then you'd set the command path by default, and only use the if block for the second case. Like in my suggestion above. I can't speak to your other link, but sorry if you're getting conflicting advice from us on that point.